### ISSUE TITLE
New German model is arrived (Kaldi_tuda_de)

### ISSUE THREAD

**Issue by KLytvynenko** [tudarmstadt-lt/kaldi-tuda-de#42]
Hi Alphacep,

I am currently using https://alphacephei.com/vosk/models.html you German model (49 Mb Lightweight wideband model for Android and RPi). It works. Accuracy is ok, but I want to know,

1. Do we have source of the scripts how you built up this model? (I mean the mobile/short version).
As I understood you built this mobile model up based on http://ltdata1.informatik.uni-hamburg.de/kaldi_tuda_de/de_400k_nnet3chain_tdnn1f_2048_sp_bi.tar.bz2

2. Kaldi_tuda_de has been provided the new extremely large model (http://ltdata1.informatik.uni-hamburg.de/kaldi_tuda_de/de_683k_nnet3chain_tdnn1f_2048_sp_bi.tar.bz2) - would it be possible me make mobile/short version for this model?

**Comment by nshmyrev** [comment]
> Hi Alphacep,

It is strange you are writing us on kaldi-tuda-de repository instead of our one.

> Do we have source of the scripts how you built up this model? (I mean the mobile/short version).
As I understood you built this mobile model up based on http://ltdata1.informatik.uni-hamburg.de/kaldi_tuda_de/de_400k_nnet3chain_tdnn1f_2048_sp_bi.tar.bz2

The small model https://alphacephei.com/kaldi/models/vosk-model-small-de-zamia-0.3.zip is from [Zamia speech ](http://zamia-speech.org/asr/). On Zamia github you can find scripts for training.

> Kaldi_tuda_de has been provided the new extremely large model (http://ltdata1.informatik.uni-hamburg.de/kaldi_tuda_de/de_683k_nnet3chain_tdnn1f_2048_sp_bi.tar.bz2) - would it be possible me make mobile/short version for this model?

This model is large and can not be used as is, but you can run the training scripts with smaller network and it should work.

**Comment by KLytvynenko** [comment]
> It is strange you are writing us on kaldi-tuda-de repository instead of our one.

Sorry, my mistake...


>  you can run the training scripts with smaller network and it should work.

Do you mean  from here https://goofy.zamia.org/zamia-speech/ ?

**Comment by bmilde** [comment]
Closing this, as this seems off topic.

I know that other people have been running de_400k_nnet3chain_tdnn1f_2048_sp_bi.tar.bz2 on Android smartphones though. If you want to train your own models you can adjust the vocabulary size, pruning factor for LM (bigger = smaller FST) and the neural network width. Tuning the filesize of the FST will give the biggest reductions.

See LM stage in https://github.com/uhh-lt/kaldi-tuda-de/blob/master/s5_r2/local/build_lm.sh for tuning the LM pruning factor.

Note that we are soon distributing even better models trained on 1000h and a new language model. You can already train this model with the updated scripts. I'll consider distributing a pruned pre-trained model this time too.

**Comment by bmilde** [comment]
Small correction, I didn't know you found the 600k model as we haven't linked or advertised it. This wasn't really meant to be distributed and also contains a bug that makes it unsuitable for online decoding. That the FST is 6GB+ large was an accident. We'll soon upload proper models trained with 1000h of audio data and a new and improved language model, as well as our own phoneme lexicon. The new models will have better LM pruning and won't be as big.

**Comment by KLytvynenko** [comment]
Great news! thank you very much for the information!! @bmilde and @nshmyrev !
